app/app.go

```diff
@@ -638,9 +639,12 @@ func (a *App) maybeAdjustRoles(ctx context.Context, cli *client.Client) error {
 again:
 	info, err := cli.Leader(ctx)
 	if err != nil {
+		a.error("failed to get leader: %v", err)
 		return err
 	}
+
 	if info.ID != a.id {
+		a.info("%s is not leader: %s", a.id, info.ID)
```

Comment:
I'd probably downgrade this log message to  debug level, using `a.debug()` instead of  `a.info()`.

Reply:
done